securedrop_client/gui/widgets.py

```diff
@@ -2007,7 +2007,7 @@ def eventFilter(self, obj, event):
         # but for reasons not entirely clear, this caused a crash. The
         # following odd way of expressing the same conditional doesn't cause a
         # crash. Go figure... :-/
-        if t == QEvent.HoverEnter or t == QEvent.HoverMove and not self.downloading:
+        if (t == QEvent.HoverEnter or t == QEvent.HoverMove) and not self.downloading:
```

Comment:
oh yeah i remember reviewing this before: https://github.com/freedomofpress/securedrop-client/pull/675#pullrequestreview-354845274

what are the details of your development machine, I don't seem to need this `HoverMove` check on my debian or qubes machine. also might you change your mind about updating this comment to be clearer or create a github issue you can link to with details on the bug?

Reply:
Sorry @creviera -- I think I mentioned I *would* update the comment, but haven't. I'll do that now and push the change. Since this *isn't* really a bug (i.e. the code works), but more a comment on why a piece of code is written in the way it is, I think an in context comment is the best place for future selves and other maintainers to find..?

Reply:
so I was going back over my first review of this code and I mentioned that `QEvent.HoverMove` should be removed all together and then this whole comment can be deleted, I believe, see my original review: https://github.com/freedomofpress/securedrop-client/pull/675#pullrequestreview-354845274

could someone test to see if just removing `HoverMove` works the way it seems to work for me?

Reply:
I'm not able to repro #791 on Ubuntu even when the code looks like this:

```python
if t == QEvent.HoverEnter and not self.downloading:
```

So I agree, not sure the `QEvent.HoverMove` is necessary here.

Reply:
Fixed in force push from Friday.